src/pages/Account/AccountGeneral.tsx

```diff
@@ -0,0 +1,146 @@
+        <Form method="POST">
+          <Row>
+            <Col md={6}>
+              <FormGroup inline>
+                <Label for={UserField.FIRST_NAME} className="form-label">First Name</Label>
+                <Input
+                  id={UserField.FIRST_NAME}
+                  invalid={!!errors.firstName && !!touched.firstName}
+                  name={UserField.FIRST_NAME}
+                  placeholder="First name"
+                  tag={Field}
+                  type="text" />
+                <FormFeedback>{errors.firstName}</FormFeedback>
+              </FormGroup>
+            </Col>
+            <Col md={6}>
+              <FormGroup inline>
+                <Label for={UserField.LAST_NAME} className="form-label">Last Name</Label>
+                <Input
+                  id={UserField.LAST_NAME}
+                  invalid={!!errors.lastName && !!touched.lastName}
+                  name={UserField.LAST_NAME}
+                  placeholder="Last name"
+                  tag={Field}
+                  type="text" />
+                <FormFeedback>{errors.lastName}</FormFeedback>
+              </FormGroup>
+            </Col>
+          </Row>
+          
+          <FormGroup>
+            <Label for={UserField.EMAIL} className="form-label">Email</Label>
+            <Input
+              disabled
+              id={UserField.EMAIL}
+              invalid={!!errors.email && !!touched.email}
+              name={UserField.EMAIL}
+              tag={Field}
+              type="email" />
+            <FormFeedback>{errors.email}</FormFeedback>
+          </FormGroup>
+
+          <FormGroup>
+            <Label for={UserField.ABOUT} className="form-label">About</Label>
+            <Textarea
+              className={`form-control${errors.about ? ' is-invalid' : ''}`}
+              html
+              name={UserField.ABOUT}
+              onBlur={() => setFieldTouched(UserField.ABOUT, true)}
+              onChange={(event: TextareaEvent) => {
+                setFieldValue(UserField.ABOUT, event.target.value);
+              }}
+              placeholder="Tell us about yourself"
+              value={values.about} />
+            <FormFeedback>{errors.about}</FormFeedback>
+          </FormGroup>
+
+          <hr />
+
+          <Row className="align-items-center justify-content-end">
+            <Col xs="auto" className="text-right float-right">
+              <Button
+                disabled={isSubmitting}
+                className="btn-success transition-3d-hover"
+                color="success"
+                onClick={submitForm}
+                type="button">
+                Save Changes
+              </Button>
+            </Col>
+          </Row>
+        </Form>
+      )}
+    </Formik>
+  );
+
+  function handleSubmit(values: any, actions: FormikActions<any>) {
+    updateUser(values)
+      .then(() => {
+        alert('user successfully updated');
```

Comment:
i think we can use reactstrap's `<Alert />` component for all of our alerts now

Reply:
implemented. It looks pretty craze-mazing